pom.xml

```diff
@@ -698,13 +698,6 @@
                                     <exclude>com.google.collections:google-collections</exclude>
                                     <!-- but not the badly numbered ones... -->
                                     <exclude>com.google.guava:guava</exclude>
-                                    <!-- Clashes with com.google.code.findbugs:annotations and having both jars -->
-                                    <!-- as a dependency then clashes with the dependency checker (because it   -->
-                                    <!-- is not very good at handling annotations). Use the annotations jar     -->
-                                    <!-- instead which has all jsr305 annotations and the additional findbugs   -->
-                                    <!-- stuff. -->
-                                    <exclude>com.google.code.findbugs:jsr305</exclude>
-                                    <exclude>net.jcip:jcip-annotations</exclude>
```

Comment:
ok that works for me, as it looks like the shade doesn't have a bunch of duplicate classes since these are test scope.  the other alternative would be to add exclusions (mostly mentioning that in case we hit this later with other projects):
```
            <dependency>
                <groupId>org.kill-bill.billing</groupId>
                <artifactId>killbill-util</artifactId>
                <version>${killbill.version}</version>
                <exclusions>
                    <exclusion>
                        <groupId>com.google.code.findbugs</groupId>
                        <artifactId>annotations</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>
```